docker-for-mac/osxfs.md

```diff
@@ -85,9 +85,11 @@ requests for ownership metadata will return the previously set
 values. Ownership-based permissions are only enforced at the OS X file
```

Comment:
The rest of the file uses "OS X" and "file system". Do you think they should be changed globally? I have no idea what to do about Apple's rename of their OS and the component is still called `osxfs` but maybe the docs should be universally updated... As for "file system", I interpret it as a noun phrase with "file" modifying "system" but I have seen lots of compound word spellings -- what is the convention in the rest of the Docker/Docker for Mac docs?